libs/sdk-core/src/breez_services.rs

```diff
@@ -227,6 +227,7 @@ impl BreezServices {
                 err: format!("Shutdown failed: {e}"),
             })?;
         *started = false;
+        self.node_api.stop().await;
```

Comment:
@ok300 This will lead to unexpected behavior if you're running the sdk on multiple clients.

Reply:
Calling `stop` even on a running client looks to be fine. The only side-effect is the next operation that uses the signer takes a bit longer, because the remote GL scheduler has to be started again.

To be safe, I agree we could remove this, e.g. not call it on `disconnect`. In that case, we can keep `stop` as a dev command and remove it from the `NodeAPI` trait, as its not meant to be called in any service call. @roeierez would you agree?